angular/headless/src/lib/coercion.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Transforms a value (typically a string) to a boolean.
+ * Intended to be used as a transform function of an input.
+ *
+ *  @usageNotes
+ *  ```typescript
+ *  @Input({ transform: auBooleanAttribute }) status: boolean | undefined;
+ *  ```
+ * @param value Value to be transformed.
+ */
+export function auBooleanAttribute(value: unknown): boolean | undefined {
+	if (value === undefined) {
+		return undefined;
+	}
+	return typeof value === 'boolean' || value === undefined ? value : value != null && value !== 'false';
```

Comment:
Why do you again compare with undefined? It is already done line 12.

Reply:
Indeed, thanks !